src/components/Dialog-Window/Notes/DialogNote.tsx

```diff
@@ -34,10 +35,23 @@ export const Note: React.FC<NoteProps> = ({ maxLength, id }) => {
     );
   };
 
+  const countWords = (): void => {
+    let count = 0;
+    const words = note.split(" ");
+    words.forEach(s => {
+      const trim = s.trim();
+      if (trim.length > 0) {
+        count += 1;
+      }
+    });
```

Comment:
Can be functionalized, but I'm not sure if it's any cleaner

```suggestion
const count = note.split(" ")
  .map(s => s.trim())
  .filter(Boolean)
  .length;
```

Reply:
I think it is cleaner, will update!